package multipleException;

import java.io.IOException;
import java.text.ParseException;

public class App {
	public static void main(String args[]){
	Multi multi1 = new Multi();
	/*try {
		multi1.run();
	} catch (IOException e) {
		// TODO Auto-generated catch block
		System.out.println(e.getMessage());
	}*/
	try {
		multi1.run();
	} catch (IOException e) {
		// TODO Auto-generated catch block
		e.printStackTrace();
	} catch (ParseException e) {
		// TODO Auto-generated catch block
		System.out.println(e.getMessage());
	}
	
	
	
	
	}

}
